.github/workflows/dists.yml

```diff
@@ -3,7 +3,7 @@ name: dists
 on:
   pull_request:
   push:
-    branches: '**'  # not tags
+    branches: [master]
```

Comment:
I do like being able to run CI on my branches without creating a PR. Does the `'**'` setting cause any problems?

Reply:
the setting causes duplicate jobs for every commit pushed to a PR. i think what you're looking for is `workflow_dispatch`. we already added it in a previous PR: it adds a Run Workflow button on the right hand side here, where you can run CI manually on any branch without creating a PR https://github.com/oconnor663/blake3-py/actions/workflows/tests.yml

Reply:
Got it.